src/snowflake/snowpark/dataframe.py

```diff
@@ -1624,6 +1633,8 @@ def pivot(
             pivot_col: The column or name of the column to use.
             values: A list of values in the column.
         """
+        if not values:
+            raise ValueError("values cannot be None or empty")
```

Comment:
This should never be `None` according to the argument's type-hint
```suggestion
            raise ValueError("values cannot be empty")
```